src/components/FlatListItem/index.js

```diff
@@ -3,34 +3,31 @@ import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
 
 import Icon from 'react-native-vector-icons/MaterialIcons';
 
-import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
+import {COLORS} from '../../utils/theme';
+import {useDynamicColors} from '../../hooks/useToggleColors';
 
-const FlatListItem = ({value, index, select, onSelect, icon, desc}) => {
+const FlatListItem = ({value, index, select, onSelect, icon, desc, isAnonimity = false}) => {
+  const dynamicColor = useDynamicColors(isAnonimity);
   if (icon) {
     return (
-      <TouchableOpacity
-        testID='onselect'
-        style={styles.container}
-        onPress={() => onSelect(index)}>
+      <TouchableOpacity testID="onselect" style={styles.container} onPress={() => onSelect(index)}>
         <View style={styles.contentText}>
           {icon}
           <View style={styles.containerLabel}>
             <Text style={styles.labelIcon}>{value}</Text>
             <Text style={styles.desc}>{desc}</Text>
           </View>
         </View>
-        {index === select && (
-          <Icon name="check-circle" color={colors.bondi_blue} size={20} />
-        )}
+        {index === select && <Icon name="check-circle" color={dynamicColor.primary} size={20} />}
       </TouchableOpacity>
     );
   }
   return (
-    <TouchableOpacity testID='noicon' style={styles.container} onPress={() => onSelect(index)}>
+    <TouchableOpacity testID="noicon" style={styles.container} onPress={() => onSelect(index)}>
       <Text style={styles.label}>{value}</Text>
       {index === select && (
-        <Icon testID='circle-icon' name="check-circle" color={colors.bondi_blue} size={20} />
+        <Icon testID="circle-icon" name="check-circle" color={dynamicColor.primary} size={20} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isAnonimity` prop is not being used in the old hunk. If this is a new feature, ensure that it has been properly tested and documented. Also, consider renaming `isAnonimity` to `isAnonymous` for better readability.



<!-- This is an auto-generated comment by OSS CodeRabbit -->